### ISSUE TITLE
AVT 1 - FileUploader (all examples) have DAP violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4221]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to  any of the React FileUploader Components
Run DAP and the following violation is present (see screenshot). This issue is present:
<img width="1248" alt="fileUploaderbutton" src="https://user-images.githubusercontent.com/21676914/66213722-f8293600-e685-11e9-977d-c6c4239decb5.png">

**Comment by emyarod** [comment]
related #3316 #3362

duplicate of https://github.com/carbon-design-system/carbon/issues/4208 ?
